t/37-ultimagen-monitor-runfolder.t

```diff
@@ -80,7 +80,8 @@ subtest 'test tracking update on new run' => sub {
   lives_ok {$test->process_run();} 'process_run succeeds';
   is( $test->tracking_run()->folder_name, $run_folder_name, 'folder_name of new tracking run' );
   is( $test->tracking_run()->flowcell_id, $ultimagen_runid, 'ultimagen_runid of new tracking run' );
-  is( $test->tracking_run()->batch_id, undef, 'undef batch_id of new tracking run' );
+  is( $test->tracking_run()->batch_id, 'O01_PRJ_batch_12345_pool_4',
```

Comment:
In this test value there is probably a typo. The first character seems to be a letter 'O'. Should it be a 3 digits number '001'?